valhalla/baldr/graphconstants.h

```diff
@@ -42,11 +42,12 @@ constexpr uint16_t kHOVAccess = 128;
 constexpr uint16_t kWheelchairAccess = 256;
 constexpr uint16_t kMopedAccess = 512;
 constexpr uint16_t kMotorcycleAccess = 1024;
+constexpr uint16_t kGolfCartAccess = 2048;
```

Comment:
Worth noting: this PR maxes out all available bit field constants, per the comment on L33 which states we have 12 bits available.